pkg/backend/stack.go

```diff
@@ -127,16 +127,53 @@ func GetStackTags() (map[apitype.StackTagName]string, error) {
 			tags[apitype.ProjectDescriptionTag] = *proj.Description
 		}
 
-		if owner, repo, err := gitutil.GetGitHubProjectForOrigin(filepath.Dir(projPath)); err == nil {
-			tags[apitype.GitHubOwnerNameTag] = owner
-			tags[apitype.GitHubRepositoryNameTag] = repo
+		// Add the git metadata to the tags.
+		if err := addGitMetadataToStackTags(tags, projPath); err != nil { // nolint
+			// Intentionally ignore errors adding git metadata to stack tags.
 		}
-
 	}
 
 	return tags, nil
 }
 
+// addGitMetadataToStackTags fetches the git repository from the directory, and attempts to detect
+// and add any relevant git metadata as stack tags.
+func addGitMetadataToStackTags(tags map[apitype.StackTagName]string, projPath string) error {
+	repo, err := gitutil.GetGitRepository(filepath.Dir(projPath))
+	if repo == nil {
+		return fmt.Errorf("no git repository found from %v", projPath)
+	}
+	if err != nil {
+		return err
+	}
+
+	remoteURL, err := gitutil.GetGitRemoteURL(repo, "origin")
+
+	if err != nil {
+		return err
+	}
+
+	if remoteURL == "" {
+		return nil
+	}
+
+	if owner, repo, kind, err := gitutil.TryGetVCSInfo(remoteURL); err == nil {
```

Comment:
Having `TryGetVCSInfo` return an error doesn't seem right. It should be the case that if we don't recognize the remote URL as a known VCS host, it isn't an error case. We just don't tag the stack in any particular way. Right?